api-reference/10 UI Components/dxHtmlEditor/3 Methods/focus().md

```diff
@@ -0,0 +1,11 @@
+---
+id: dxHtmlEditor.focus()
+---
+---
+##### shortDescription
+Sets focus on the content field of the editor.
+
+---
+<!-- %fullDescription% -->
+
+<!-- import * from 'api-reference\10 UI Components\dxTextEditor\3 Methods\focus().md' -->
```

Comment:
The full description will be inherited from `Widget` using the default inheritance mechanism.

```suggestion
```